llm/gpt-2/gpt2-train.yaml

```diff
@@ -0,0 +1,89 @@
+name: train
+
+envs:
+  BUCKET_NAME: # Fill in your bucket name
+
+resources:
+  accelerators: A100:8
+  # Use docker image for latest version g++ to enable the compilation of llm.c.
+  image_id: docker:nvidia/cuda:12.4.1-cudnn-devel-ubuntu22.04
+  image_id:
+    # Avoid using docker image for lambda due to the docker is not supported on
+    # Lambda yet, but the base image works.
+    - cloud: lambda
+      image_id: null
+    - cloud: aws
+    - cloud: gcp
+    - cloud: azure
+    - cloud: fluidstack
+  
+file_mounts:
+  ~/.cache/huggingface: gs://$BUCKET_NAME
```

Comment:
This syntax puts a dependency  on bucket being uploaded to gcs (`gs://`). Maybe lets use the (name, mode) syntax (commented below) which is agnostic of the store used to upload?